site/content/en/docs/getting_started/overview.md

```diff
@@ -0,0 +1,202 @@
+---
+title: 'CVAT Overview'
+linkTitle: 'Introduction to CVAT'
+weight: 1
+description: 'CVAT: The open-source tool for image and video annotation and data labeling'
+---
+
+Machine learning systems often struggle due to
+poor-quality data. Without effective tools,
+improving a model can be tough and inefficient.
+
+[**CVAT.ai**](https://www.cvat.ai/) **is a** versatile **tool** for
+**annotating images and videos**, serving the computer
+vision community worldwide.
+
+Our goal is to help developers, businesses,
+and organizations globally by using a Data-centric AI approach.
+
+CVAT offers two versions:
+
+- [**CVAT Cloud**](https://app.cvat.ai/): Start online with CVAT,
+  **available for free**. You can also choose a [**subscription**](https://www.cvat.ai/pricing/cloud)
+  for **unlimited data**, **organizational tools**, **auto-annotations**, and [**more**](https://www.cvat.ai/post/cvat-ai-pricing-plans-choosing-the-right-plan-for-your-needs).
+
+- **Self-hosted CVAT** (forever Free of charge): Follow the [**Self-hosted Installation Guide**](/docs/administration/basics/installation/)
+  for setup. We provide [**Enterprise-level support**](https://www.cvat.ai/pricing/on-prem)
+  for this version, **including premium features** like **SSO**, **LDAP**, advanced integrations with
+  [**Roboflow and HuggingFace**](https://www.cvat.ai/post/integrating-hugging-face-and-roboflow-models),
+  and **advanced analytics**. We also offer **professional training** and **24-hour SLA support**.
+
+
+See:
+
+- [Tools and formats](#tools-and-formats)
+  - [Supported formats](#supported-formats)
+  - [Annotation tools](#annotation-tools)
+- [Automated labeling](#automated-labeling)
+- [Useful links](#useful-links)
+- [Integrations](#integrations)
+- [License Information](#license-information)
+- [Get in touch](#get-in-touch)
+
+## Tools and formats
+
+CVAT stands as a comprehensive
```

Comment:
Could you please fix formatings? I see a couple of words on each line. Why?

Reply:
@nmanovic I do not see why the formatting must be fixed. It looks ok for me. You see a couple of words on each line, because I cut them this way for the linter not to throw an error for the lines to be too long. Docsy does not see it as "cut"  anyway and the final version looks fine. 
![image](https://github.com/opencv/cvat/assets/39969264/cc18a3c6-c652-46d3-89ac-5da154f17b11)

Reply:
Why don't one word per line in this way? Please make it appropriate. The recommended size of a line is around 80 characters (I believe our linter complains when it is 120). It is weird when the first line is only 30 without any reason. It is just UNREADABLE.